[WIP] #110 Add base functionality for adding a new meeting

detri opened the pull request:
**Related Issue**
https://github.com/columbus-elixir/marketing/issues/110

This PR adds a screen for adding a new meeting, as well as a few controller actions to make it possible.

**Validation**

- Pull down this PR
- Migrate (you may need to re-seed -- not sure of the implications of this on production)
- Open up your trusty http://localhost:4000/meetings/new
- Try to add a new meeting, hopefully you are successful

-----

This is my first Elixir PR! Let me know if anything needs added/changed.

detri committed:
Add base functionality for adding a new meeting

detri commented on lib/cbus_elixir_web/templates/meeting/form.html.eex:
```diff
@@ -0,0 +1,35 @@
+    <div class="column" style="flex-direction: row;">
+      <div class="form-group date-picker">
+        <%= label f, :date, class: "control-label" %>
+        <%= date_select f, :date, class: "form-control" %>
```
Note that date_select isn't compatible with utc_datetime. I figure we probably don't need to store the time anyways, and it looks like we aren't in the seeds, so I figured the migration was the best thing to do. Let me know if I'm wrong.

croz1007 commented:
@detri Do you have screenshots of this work? Are there tests for the work? What is the URL to make this function? I don't see any updates to the router and I don't see any controls for creating or editing meetings

detri commented:
> @detri Do you have screenshots of this work? Are there tests for the work? What is the URL to make this function? I don't see any updates to the router and I don't see any controls for creating or editing meetings

I'll add tests, screenshots, and an edit page. I'll also make sure the router is up to date. This is MVP right now based on the issue description, but the validation steps should be working. I can mark it as a WIP until I add all of that stuff - I appreciate all the feedback.

croz1007 commented:
@detri Sounds great. Please let me know if you have questions about the issue or have suggestions for how this can be better. Thanks for picking up the issue.

detri committed:
Put the CUD in CRUD for meetings, add tests

detri commented:
I've added tests and create/update/delete ability for meetings. It looks like the "show" page (which has the edit/delete controls) can't currently be accessed without typing in the URL manually (which means knowing the meeting ID). Same for the "new" page, you'd need to type the URL manually. Should we add a "new meeting" button to the top of the "index" page, and "show meeting" buttons on all of the meetings? Or should we create some sort of admin page for meetings separate from the current meeting pages? Let me know what you think.

Once we figure that out I'll add screenshots and then we can fix it up as needed :)

P.S. We use the `resources` function with the meetings controller in the router, meaning all of the new actions I've added to the meetings controller should already be exposed.
